logs_monitoring_elb.tf

```diff
@@ -25,6 +25,7 @@ locals {
   elb_logs_s3_bucket = "${var.elb_logs_bucket_prefix}-${var.account_name}-elb-logs"
 }
 
+#tfsec:ignore:aws-s3-block-public-acls tfsec:ignore:aws-s3-block-public-policy tfsec:ignore:aws-s3-ignore-public-acls tfsec:ignore:aws-s3-no-public-buckets tfsec:ignore:aws-s3-encryption-customer-key tfsec:ignore:aws-s3-enable-bucket-logging tfsec:ignore:aws-s3-enable-versioning tfsec:ignore:aws-s3-specify-public-access-block -- Ignore error for not having a public ACL
```

Comment:
@DJKaswa 
These logs should not be public, correct?  If they should not be public, we need to implement a proper s3 public access block resource so tfsec can run normally.